RadarSDK/RadarSettings.m

```diff
@@ -243,9 +247,15 @@ + (void)setSdkConfiguration:(RadarSdkConfiguration *)sdkConfiguration {
                             [RadarUtils dictionaryToJson:[sdkConfiguration dictionaryValue]]]];
     if (sdkConfiguration) {
         [[NSUserDefaults standardUserDefaults] setInteger:(int)sdkConfiguration.logLevel forKey:kLogLevel];
+        [[NSUserDefaults standardUserDefaults] setObject:[sdkConfiguration dictionaryValue] forKey:kSdkConfiguration];
     }
 }
 
++ (RadarSdkConfiguration *_Nullable)sdkConfiguration {
+    NSDictionary *sdkConfigurationDict = [[NSUserDefaults standardUserDefaults] dictionaryForKey:kSdkConfiguration];
```

Comment:
Is this a safe operation?

Reply:
will return nil if the value doesn't exist, then RadarSdkConfiguration from dictionary also returns nil